File: b/soccerTeam.py
class soccerTeam():
	__name = ""
	__nation = ""
	__year_founded = 0
	__player = []

	def __init__(self, name="", nation="", year_founded=0, player=[]):
		self.__name = name
		self.__nation = nation
		self.__year_founded = year_founded
		self.__player = player

	def setName(self, name):
		self.__name = name

	def getName(self):
		return self.__name

	def setNation(self, nation):
		self.__nation = nation

	def getNation(self):
		return self.__nation

	def setYear_founded(self, year_founded):
		self.__year_founded = year_founded

	def getYear_founded(self):
		return self.__year_founded

	def setPlayer(self, player):
		self.__player = player

	def getPlayer(self):
		return self.__player

	def printTeam(self):
		print("\nNama tim : ", self.__name)
		print("Negara asal tim : ", self.__nation)
		print("Tahun berdiri ", self.__year_founded)
		print("Pemain :")
		for i in self.__player:
			print(" -> ", i)

File: b/soccerTeam.php
<?php

class soccerTeam
{
	private $name = "";
	private $nation = "";
	private $year_founded = 0;
	private $player = [];

	public function __construct($name="", $nation="", $year_founded=0, $player=[])
	{
		$this->name = $name;
		$this->nation = $nation;
		$this->year_founded = $year_founded;
		$this->player = $player;
	}

	public function setName($name)
	{
		$this->name = $name;
	}

	public function getName()
	{
		return $this->name;
	}

	public function setNation($nation)
	{
		$this->nation = $nation;
	}

	public function getNation()
	{
		return $this->nation;
	}

	public function setYear_founded($year_founded)
	{
		$this->year_founded = $year_founded;
	}

	public function getYear_founded()
	{
		return $this->year_founded;
	}

	public function setPlayer($player)
	{
		$this->player = $player;
	}

	public function getPlayer()
	{
		return $this->player;
	}

	public function printTeam()
	{
		echo "Nama tim : ". $this->name ."<br>";
		echo "Negara asal tim : ". $this->nation ."<br>";
		echo "Tahun berdiri : ". $this->year_founded ."<br>";
		echo "Pemain : <br>";
		foreach($this->player as $i)
		{
			echo " -> ". $i ."<br>";
		}
	}
}

?>